### PR TITLE
Do not set single user in constructor of encryption occ commands

### LINKED ISSUE #19141: singleuser mode cannot be turned off
Something has changes the behaviour of ownCloud when installed on Travis and the instance is now in singleuser mode and integration tests fail with a 503.

I've noticed this in the VM as well. The instance stayed in singleusermode after a successful installation.

@DeepDiver1975 @nickvergessen @schiesbn

### PR DESCRIPTION
**Pull request by nickvergessen**
@schiesbn @oparoz @Xenopathic 

Fix #19141

### CODE REVIEW

**Comment by karlitschek** [comment]
good :+1:

File: core/command/encryption/encryptall.php [hunk 1]
```diff
@@ -104,8 +113,12 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		$output->writeln('');
 		$question = new ConfirmationQuestion('Do you really want to continue? (y/n) ', false);
 		if ($this->questionHelper->ask($input, $output, $question)) {
+			$this->forceSingleUserAndTrashbin();
+
 			$defaultModule = $this->encryptionManager->getEncryptionModule();
 			$defaultModule->encryptAll($input, $output);
+
+			$this->resetSingleUserAndTrashbin();
```
Comment (Xenopathic) [inline core/command/encryption/encryptall.php]: What happens if an exception is thrown during `encryptAll()`? Unlike when `__destruct()` was being used to call the cleanup logic, now it won't be run unless the operation is successful.

Comment (nickvergessen) [inline core/command/encryption/encryptall.php]: DecryptAll has a try-catch, encrypt does not, so I think it doesn't throw exceptions?

Comment (Xenopathic) [inline core/command/encryption/encryptall.php]: I'm not qualified to comment on that, but IMHO any function has a chance of throwing an exception, and the cleanup logic must run if that happens

**Comment by oparoz** [comment]
Fix works: https://travis-ci.org/owncloud/gallery/builds/80985991, but I agree with @Xenopathic, it's very important to try and catch errors in encryptAll() and to cleanup in case they show up. So many bad things can happen when reading files.

File: core/command/encryption/encryptall.php (hunk 1 above)
Comment (nickvergessen) [inline core/command/encryption/encryptall.php]: added

**Comment by Xenopathic** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**